### ISSUE TITLE
Instrument with accessibility

### ISSUE THREAD

**Issue by zepumph** [phetsims/molarity#57]
Here is the link to the design doc. https://docs.google.com/document/d/1Z_5biEOBk9W4lVPXxGt7fyg-V6_CwTA4vMZ0mVMTJsI/edit#heading=h.nhqxjbby3dgu

**Comment by zepumph** [comment]
As we begin to add accessible instrumentation to this sim, it would be good to keep track of it in this issue.

**Comment by pixelzoom** [comment]
And good to involve the responsible developer!

**Comment by zepumph** [comment]
Indeed! @pixelzoom, @jessegreenberg, @twant, and @zepumph will be adding features in the coming months. I think that it will loosely follow something like:

* Implement PDOM like labels and help text
* Add the screen summary
* Add dynamic updates to the PDOM that are required.
* Add dynamic alerts and aria-valuetext.

I will link to the design doc when we get a permanent link to it.

**Comment by pixelzoom** [comment]
- [ ] When this issue is completed, let's plan to give me an overview of what has been done and why.  I've been largely out of the loop, which is fine.  But as the responsible developer, I should be able to maintain this sim going forward, hopefully without having to involve the a11y team, or at least knowing when I need to consult the a11y team.

**Comment by pixelzoom** [comment]
@zepumph said:
> I will link to the design doc when we get a permanent link to it.

Does this link exist yet?

**Comment by zepumph** [comment]
> When this issue is completed, let's plan to give me an overview of what has been done and why. I've been largely out of the loop, which is fine. But as the responsible developer, I should be able to maintain this sim going forward, hopefully without having to involve the a11y team, or at least knowing when I need to consult the a11y team.

That sounds really nice. I also think that doing that at some point 50% (ish) through would be good to. I foresee this being more of dialog/review than just an explanation.

Here is the link to the new design doc that was recently created. https://docs.google.com/document/d/1Z_5biEOBk9W4lVPXxGt7fyg-V6_CwTA4vMZ0mVMTJsI/edit#heading=h.nhqxjbby3dgu

I'll add it up top too.

**Comment by twant** [comment]
> * [ ]  When this issue is completed, let's plan to give me an overview of what has been done and why.  I've been largely out of the loop, which is fine.  But as the responsible developer, I should be able to maintain this sim going forward, hopefully without having to involve the a11y team, or at least knowing when I need to consult the a11y team.

Sounds great. I'll leave it up to @zepumph to decide the best timing to take a look at the a11y features together, as I don't really have a good sense of how far through the process of adding a11y features we actually are. Looking forward to discussing with you!

**Comment by zepumph** [comment]
This issue seems too broad and in general outdated. @twant would you like to keep this open?

**Comment by twant** [comment]
I think it makes sense to close it, given the more specific issues. closing now!
